### PR TITLE
wrote some tests for todo component

### PR DESCRIPTION
**Pull request by vinayak-trivedi**
solves #134. Added some tests for todo component. It does not contain tests for features implemented using React native gesture handler.

**this PR should merge after merging** #144

### CODE REVIEW

**Review by pranikz — APPROVED** [review]
LGTM

**Review by pranikz — DISMISSED** [review]

File: src/components/ToDoComponent/Card.tsx [hunk 1]
```diff
@@ -107,7 +107,7 @@ const Card = ({ item, posStyle, changecard, removeCard }: props) => {
                 )}
               </View>
               <View style={CardStyles.flex}>
-                <TouchableOpacity onPress={markDone}>
+                <TouchableOpacity onPress={markDone} testID='doneBtn'>
                   {checked ? (
```
Comment (bajajcodes) [inline src/components/ToDoComponent/Card.tsx]: Terenary operator can be used inside `source` prop of `Image` component
```js
source={checked ? Images.checkedIcon : Images.uncheckedIcon}
```

**Review by bajajcodes — COMMENTED** [review]
- Code is good to merge

File: __tests__/Component-test.tsx [hunk 2]
```diff
@@ -16,3 +19,44 @@ test('flatlist exists when we click on the arrow button', () => {
     expect(queryByTestId('flatlist')).toBeTruthy();
     expect(getByTestId('arrowBtnIcon').props.style).toStrictEqual({resizeMode: 'cover', width: 30, height: 30, transform: [{ rotate: '90deg' }],})
 })
+
+// ToDoComponent Tests
+
+jest.mock('react-native-gesture-handler', () => {})
```
Comment (RitikJaiswal75) [inline __tests__/Component-test.tsx:25]: do we need this line if we have something similar in line 27

Comment (vinayak-trivedi) [reply __tests__/Component-test.tsx:25]: removing this leads to test failing

File: src/components/ToDoComponent/Card.tsx (hunk 1 above)
Comment (vinayak-trivedi) [reply src/components/ToDoComponent/Card.tsx]: yes makes sense, I will change it

**Review by shreya-mishra — APPROVED** [review]
LGTM

File: src/components/ToDoComponent/Card.tsx (hunk 1 above)
Comment (shreya-mishra) [reply src/components/ToDoComponent/Card.tsx]: Change done ?

Comment (vinayak-trivedi) [reply src/components/ToDoComponent/Card.tsx]: yes

**Pull request merged**